openlibrary/catalog/marc/parse.py

```diff
@@ -553,13 +527,14 @@ def read_contributions(rec):
     :param (MarcBinary | MarcXml) rec:
     :rtype: dict
     """
+
```

Comment:
Add $0 & $1 for 700, 710, 711

Reply:
@tfmorris This is interesting, I didn't know about these. I'm not sure exactly where to extract these ids to in OL's data model.

Both `$0` and `$1` are repeatable, so there could be multiple values. It also seems like the ids and URIs could be _anything_ in general.

Existing test data only has `7xx$0` examples:

```
700 1  $6 880-05 $a El Moudden, Abderrahmane. $0 http://id.loc.gov/authorities/names/nr92001540
700 1  $6 880-06 $a Bin-Ḥāddah, ʻAbd al-Raḥīm. $0 http://id.loc.gov/authorities/names/nr97026593
700 1  $6 880-07 $a Gharbi, Mohamed Lazhar. $0 http://id.loc.gov/authorities/names/nr96019749
710 2  $6 880-08 $a Jāmiʻat Muḥammad al-Khāmis. $b Kullīyat al-Ādāb wa-al-ʻUlūm al-Insānīyah. $0 http://id.loc.gov/authorities/names/n83213755
700 1  $6 880-04 $a Hayashiya, Tatsusaburō, $d 1914-1998. $0 http://id.loc.gov/authorities/names/n81047233
700 1  $6 880-05 $a Yokoi, Kiyoshi. $0 http://id.loc.gov/authorities/names/n81089234
700 1  $6 880-06 $a Narabayashi, Tadao, $d 1940-1960. $0 http://id.loc.gov/authorities/names/n85206624
```

They all happen to be LOC URIs. It looks like we might be able to assume all URIs in this field will be valid web URLs, but I'm not sure that is strictly a given. 

I'm thinking adding and making use of these $0 / $1 author sub-fields should be a separate feature (building on this one).

Reply:
New issue raised https://github.com/internetarchive/openlibrary/issues/7724